web/main.ts

```diff
@@ -2490,11 +2523,23 @@ class GitGraphView {
 					});
 					const commitDetails = expandedCommit.commitDetails!;
 					const parents = commitDetails.parents.length > 0
-						? commitDetails.parents.map((parent) => {
+						? commitDetails.parents.map((parent, parentIndex) => {
 							const escapedParent = escapeHtml(parent);
-							return typeof this.commitLookup[parent] === 'number'
-								? '<span class="' + CLASS_INTERNAL_URL + '" data-type="commit" data-value="' + escapedParent + '" tabindex="-1">' + escapedParent + '</span>'
-								: escapedParent;
+
+							const parentWasLoaded = loadedParents.includes(parent);
+							const isComparedToParent = parentIndex + 1 === expandedCommit.parentIndex;
+
+							let parentHtml = abbrevCommit(escapedParent);
+							if(parentWasLoaded) {
+								parentHtml = `<span class="${CLASS_INTERNAL_URL} ${CLASS_PARENT}-${parentIndex + 1}" data-type="commit" data-value="${escapedParent}" tabindex="-1">
+									${parentHtml}
+								</span>`;

```

Comment:
The multi-line string (2534-2536) is causing whitespace to be introduced after the commit hash `9c7eb8ce`:
![image](https://user-images.githubusercontent.com/5315753/115528594-8818ea00-a2d5-11eb-9487-477113c5913b.png)